big-red-shuttle/EmergencyViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  EmergencyViewController.swift
+//  big-red-shuttle
+//
+//  Created by Austin Astorga on 10/26/16.
+//  Copyright © 2016 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class EmergencyViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    var tableView: UITableView!
+    let emergencyArray = [("Call Cornell Univeristy Police", 6072551111), ("Call Emergency Services", 911), ("Call Blue Light Escorts", 6072557373), ("Call Gannett Health Services", 6072555155)] //add more later
```

Comment:
That would be better, but this is still a code reusability hazard. What if you decide you need to add an email to these entries? Or import all this information from a JSON source. Tuples just aren't great for representing an idea, but the naming would definitely be an improvement. Consider what I say for future if you like.